packages/calcite-components/src/components/block-section/block-section.scss

```diff
@@ -4,6 +4,16 @@
   color: var(--calcite-color-text-2);
 }
 
+calcite-switch {
+  // --calcite-switch-background-color: var(--calcite-block-section-switch-background-color);
+  // --calcite-switch-border-color: var(--calcite-block-section-switch-border-color);
+  // --calcite-switch-corner-radius: var(--calcite-block-section-switch-corner-radius);
+  // --calcite-switch-handle-background-color: var(--calcite-block-section-switch-handle-background-color);
+  // --calcite-switch-handle-border-color: var(--calcite-block-section-switch-handle-border-color);
+  // --calcite-switch-handle-corner-radius: var(--calcite-block-section-switch-handle-corner-radius);
```

Comment:
Do we need both corner-radius props or can we apply a single to both the handle and track?

Reply:
> Do we need both corner-radius props or can we apply a single to both the handle and track?

Probably not, ill remove and use the other var.